docs/README.md

```diff
@@ -64,7 +64,7 @@ $  mvn clean install
 
 You could expect to see _BUILD SUCCESS_ at the end of the build console.
 
-![Alcor Build](modules/ROOT/images/alcor_build.JPG)
+<img src="modules/ROOT/images/alcor_build.png" alt="Alcor Build" width="720">
```

Comment:
@pkommoju This was Yan's check in yesterday. It shows up here likely because the nmm_ncm_api_02 branch is not in sync with the master branch. Possibly make your master branch up to date and then switch to feature branch and do git merge master.

Reply:
Yes, this is from Yan's checkin. I am going in circles with merge conflicts. I just started a clean rebuild. Let me see what happened to it.

Reply:
when I syncup my branch, I am expected to get Yan's chekin, right?